Doc/whatsnew/3.8.rst

```diff
@@ -181,6 +181,9 @@ Other Language Changes
   and Windows use this to properly terminate scripts in interactive sessions.
   (Contributed by Google via Gregory P. Smith in :issue:`1054041`.)
 
+* fcntl now has File Sealing operations.(Contributed by Joannah Nanjekye
```

Comment:
Please, use markdown formatting for fcntl (:func:`fcntl.fcntl`).

Reply:
What about write the final `.` after "()" And don't write the `.` inside the parenthesis. Like this:

```
fcntl now has File Sealing operations (Contributed by Joannah Nanjekye in :issue:`26835`).
```

Reply:
Conform to the style used in other entries.